src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,157 @@
+footer {
+  font-size: 0.875rem;
+  background: $gray-dark;
+  color: transparentize($white,0.5);
+  padding-top: 50px;
+  padding-bottom: 50px;
+}
+
+.footer-links {
+  text-align: center;
+  list-style-type: none;
+  padding-left: 0;
+  margin-bottom: 0;
+
```

Comment:
This functionality is possible with the grid in Bootstrap v4.alpha6. I'll work on a PR now to bump the version we're using to that so that this can be done with less CSS.